library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -44,6 +68,24 @@ PlatformBridgeFilter::PlatformBridgeFilter(PlatformBridgeFilterConfigSharedPtr c
   ASSERT(platform_filter_.instance_context,
          fmt::format("init_filter unsuccessful for {}", filter_name_));
   iteration_state_ = IterationState::Ongoing;
+
+  if (platform_filter_.set_request_callbacks) {
+    platform_request_callbacks_.resume_iteration = envoy_filter_callback_resume_decoding;
+    platform_request_callbacks_.release_callbacks = envoy_filter_release_callbacks;
+    platform_request_callbacks_.callback_context =
+        new PlatformBridgeFilterWeakPtr{weak_from_this()};
```

Comment:
we should have an inline comment about the ownership model

Reply:
Sure.